lib/modules/styleTweaks.js

```diff
@@ -171,6 +171,23 @@ module.options = {
 		description: 'styleTweaksHideDomainLinkDesc',
 		bodyClass: true,
 	},
+	flairEmojiAsText: {
+		title: 'styleTweaksflairEmojiAsText',
+		description: 'styleTweaksflairEmojiAsTextDesc',
+		type: 'enum',
+		value: 'never',
+		values: [{
+			name: 'Never',
```

Comment:
Could you also move the `name` values into `en.json` with (gnarly long) keys like `styleTweaksflairEmojiAsText`?  Then you can load them in here as `i18n('styleTweaksflairEmojiAsTextNever')`

Reply:
Oh, yes, this sounds right.